src/components/NavBar/index.js

```diff
@@ -43,7 +56,11 @@ export default class NavBar extends Component {
         <Collapse in={!ui.searchBarMode}>
           <Toolbar className={classes.navBarWrapper}>
             <div className={classes.navSection}>
-              <BodhiLogo {...this.props} />
+              <BodhiLogo {...this.props} onClick={() => this.handleClickLogo()} />
+              {
+                // eslint-disable-next-line
```

Comment:
is the audio class not allowed?

Reply:
will have this error: Media elements such as <audio> and <video> must have a <track> for captions